Caddyfile

```diff
@@ -0,0 +1,3 @@
+{$DOMAIN_NAME} {
+    reverse_proxy server:8080
```

Comment:
The port should also be fetched from the environment because it could change in the server based on the PORT env

Reply:
Shall fix